### DISCUSSION TITLE
Server-side authentication for interaction via our CLI

### DISCUSSION THREAD

**Discussion by jaredcosulich** [supabase/supabase#5771] [Questions]
We're developing a CLI that interacts with our website. We need to allow users to interact with the Supabase database with row level security from the CLI. I was thinking the user could go through our server to talk to Supabase, but I can't find any way to authenticate the user from the server.

Is there any way to log a user in from the server such that we can make queries on behalf of the user (and get through RLS) and return the results to the CLI (our CLI the user is using to interact with our website, not the Supabase CLI)?

**Comment by GaryAustin1** [comment]
Here are a couple of threads on "logging" in from a server as the user with a jwt so RLS works.
These assume your users do have accounts already on supabase.

https://github.com/supabase/supabase/discussions/1094
https://github.com/supabase/supabase/issues/491
